### PR TITLE
Feature/lsi 194 endpoint for bulk user creation

### PR DESCRIPTION
**Pull request by jeethufoc**
This PR is related to an adding endpoint for bulk user creation

### CODE REVIEW

File: tests/Traits/FileRemovalTrait.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+namespace OAT\SimpleRoster\Tests\Traits;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+trait FileRemovalTrait
+{
+    protected function removeGeneratedUsersFilePath(): void
+    {
+        $filesystem = new Filesystem();
+        $filesystem->remove($_ENV['GENERATED_USERS_FILE_PATH']);
```
Comment (SergiiTao) [inline tests/Traits/FileRemovalTrait.php:32]: I don't think this is a good approach to using ENV vars directly.

File: tests/Traits/FileRemovalTrait.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ *  Copyright (c) 2021 (original work) Open Assessment Technologies S.A.
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\Tests\Traits;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+trait FileRemovalTrait
```
Comment (SergiiTao) [inline tests/Traits/FileRemovalTrait.php:27]: where it is used?

**Review by SergiiTao — CHANGES_REQUESTED** [review]

File: tests/Traits/FileRemovalTrait.php (hunk 2 above)
Comment (jeethufoc) [reply tests/Traits/FileRemovalTrait.php:27]: It is used in BulkCreateUserActionTest to auto remove all the test files generated before test case execution starts. Previously I missed out to add it in BulkCreateUserActionTest. Now its added

File: src/DataTransferObject/UserCreationResultMessage.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            $userPrefixString
+        );
+    }
+
+    private function multipleSlugNormalizeMessage(array $slugTotalUsers, string $userPrefixString): string
+    {
+        $message = '';
+        foreach ($slugTotalUsers as $slug => $totalUsers) {
+            $message .= sprintf(
+                "%s users created for line item %s for user prefix %s \n",
```
Comment (SergiiTao) [inline src/DataTransferObject/UserCreationResultMessage.php:51]: double quotes

**Review by SergiiTao — APPROVED** [review]

**Review by vraginya — APPROVED** [review]
Great job.

**Pull request merged**